bundle/2.0.0/manifests/skupper-operator.v2.0.0.clusterserviceversion.yaml

```diff
@@ -330,31 +485,26 @@ spec:
   - van
   links:
   - name: Skupper Operator
-    url: https://github.com/skupperproject/skupper-operator
+    url: https://github.com/ajssmith/skupper-operator
   maintainers:
   - email: skupper@googlegroups.com
     name: Skupper Community
   maturity: alpha
+  minKubeVersion: 1.25.0
   provider:
     name: Skupper Project
     url: https://skupper.io
   relatedImages:
-  - image: quay.io/skupper/site-controller@sha256:461c3820f35e55fbc0f9bb6fac9ec8cc493b976dc69eed69eb28624e8cb446b6
-    name: skupper-site-controller
-  - image: quay.io/skupper/skupper-router@sha256:a1c07d18a45377d307621b401fb1c86570379ab2d6603fcb1ef86857f4b2f45b
+  - image: quay.io/skupper/controller@sha256:c1b0070551c050fa9069e722690654f6a519b523c4b616c0ce2e5421d489cfac
+    name: skupper-controller
+  - image: quay.io/skupper/skupper-router@sha256:416de2e5971058c67cbf2bbc0cdbad6ef234dbc7617d2a8ba528a3dd96747e39
     name: skupper-router
-  - image: quay.io/skupper/service-controller@sha256:74a5ff88666af51d577c341c625d204eca6609b882783bf03e91333eb1bc1c2f
-    name: skupper-service-controller
-  - image: quay.io/skupper/config-sync@sha256:0002f4fd396d2aa94f51403974ae6b77453c0bc97872ff78edd6a38033745705
-    name: skupper-config-sync
-  - image: quay.io/skupper/flow-collector@sha256:db5b25bd00b5c4b25f11a2bbbdf758fe1faf7f99e12a71e5975820d70b400511
-    name: skupper-flow-collector
+  - image: quay.io/skupper/kube-adaptor@sha256:b83a428099a00d7e3163fdc09683fbf15209feb1bad49defbb86b9c2fa6f28bd
+    name: skupper-kube-adaptor
+  - image: quay.io/skupper/network-observer@sha256:d2ab0a27783fd4ad96a8853e2847b99a0be0043687b8a5d1ebfb2dd3fa4fd1b8
+    name: skupper-network-observer    
```

Comment:
Network observer now uses: mirror.gcr.io/nginxinc/nginx-unprivileged
We might need to consider adding it here as well.